pkg/plugins/policies/meshaccesslog/api/v1alpha1/meshaccesslog.go

```diff
@@ -121,11 +121,12 @@ type Format struct {
 	Plain *string `json:"plain,omitempty"`
 	// +kubebuilder:example={{key: "start_time", value: "%START_TIME%"},{key: "bytes_received", value: "%BYTES_RECEIVED%"}}
 	Json *[]JsonValue `json:"json,omitempty"`
+	// +kubebuilder:validation:Optional
```

Comment:
could you please somehow mark this as workaround? when we implement Universal support for `+kubebuilder:default` we can remove `Optional` right?

Reply:
Or maybe it's fine as it is even long-term. Should we have a rule that all `+kubebuilder:default` always comes with `+kubebuilder:validation:Optional`?

Reply:
We won't need this workaround after this https://github.com/kumahq/kuma/pull/12829

Reply:
> Or maybe it's fine as it is even long-term. Should we have a rule that all +kubebuilder:default always comes with +kubebuilder:validation:Optional?

already is there https://github.com/kumahq/kuma/blob/ef0a835d5a11896edc749ac0b25b7db1b993bf07/tools/ci/api-linter/linter/api-linter.go#L193